src/main/java/com/apigee/edge/config/mavenplugin/UserRoleMojo.java

```diff
@@ -218,7 +218,7 @@ public static String createUserRole(ServerProfile profile, String userRole) thro
 		UserRole userRoleObject = gson.fromJson(userRole, UserRole.class);
 		String namePayload = userRoleConversion(userRoleObject.name);
 
-		HttpResponse response = RestUtil.createOrgConfig(profile, userRole, namePayload);
+		HttpResponse response = RestUtil.createOrgConfig(profile, UserRoleMojo.userRole, namePayload);
```

Comment:
This looks good.. Please check line 250 and 280 in this file. Not sure what is the second argument being passed. It shuld be resource path in the Mgmt API. Please check other Mojo classes

There is reference to userRole but its being passed.

Reply:
@ssvaidyanathan  Thanks for the review. 
The second argument being passed is resource path that is "userroles" string which we assign in class level variable userRole.

Reply:
Did you check line 250 and 280 as I mentioned ? Can you update it to `UserRoleMojo.userRole`

Reply:
Its ok - I will make the changes